x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -209,8 +250,20 @@ export const ConfigurationStepForm: FC<CreateAnalyticsStepProps> = ({
     if (jobTypeChanged) {
       setLoadingFieldOptions(true);
     }
-
-    const { success, expectedMemory, fieldSelection, errorMessage } = await fetchExplainData(form);
+    // Ensure runtime field is in 'includes' table if it is set as dependent variable
+    const formCopy = cloneDeep(form);
+    const depVarIsRuntimeField =
+      runtimeMappings &&
+      Object.keys(runtimeMappings).includes(dependentVariable) &&
+      includes.length > 0;
+    const unique = depVarIsRuntimeField
+      ? Array.from(new Set([...includes, dependentVariable]))
+      : undefined;
+    formCopy.includes = unique ? unique : formCopy.includes;
+
+    const { success, expectedMemory, fieldSelection, errorMessage } = await fetchExplainData(
```

Comment:
do we need a `try...catch`?

Reply:
`fetchExplainData` wraps the actual call in a try/catch and returns the errorMessage if present. 👍